packages/expo/src/Expo.ts

```diff
@@ -92,7 +92,7 @@ export { Audio };
 export { default as Video } from './av/Video';
 export { default as BlurView } from './effects/BlurView';
 export { default as LinearGradient } from './effects/LinearGradient';
-import * as FacebookAds from './facebook-ads';
+import * as FacebookAds from './facebook-ads/index';
```

Comment:
Shouldn't need this...

Reply:
I did though :/

Reply:
OK. At minimum we should add a comment or rename index.js because this differs from the typical behavior of index.js, but if you could figure out what is breaking so we can fix it at the source, that'd be even better.